src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -347,6 +348,22 @@ public T WithB2CAuthority(string authorityUri)
             return this as T;
         }
 
+        /// <summary>
+        /// Specifies additional parameters acquired from authentication responses to be cached with the access token that are normally not included in the cache object.

```

Comment:
Add a comment on where these values can be read from. And a comment about results from cache.